source/upgrade.txt

```diff
@@ -145,7 +145,14 @@ The v4.1 driver introduces the following breaking changes to the ABI:
 
   .. note::
 
-     This does not affect users who compile with C++17 or newer and have not set ``BSONCXX_POLY_USE_IMPLS=ON``
+     This does not affect users who compile with C++17 or newer and have not set
+     ``BSONCXX_POLY_USE_IMPLS=ON``
+
+.. _version-4.1-enable-tests-config:
```

Comment:
S: I don't think you can add ref anchors within a section, so looks like this one just goes to :ref:`version-4.1-build-system-breaking`. you can remove it and replace reference to it with the general 4.1 breaking changes ref